service/systemd/service.go

```diff
@@ -105,7 +111,7 @@ type dbusAPI interface {
 	Reload() error
 }
 
-var newConn = func() (dbusAPI, error) {
+var NewDBusAPI = func() (DBusAPI, error) {
```

Comment:
Do we really need this now?

Reply:
We could do away with the factory, pass an instance and do away with the per-use instantiation. It would be nicer.

Reply:
Except that calling ```dbus.New()``` opens a connection, which is closed after each use. We would be holding this open for the duration of the service instance life, which might not be so great.